core/app/service/setting.go

```diff
@@ -208,20 +222,6 @@ func (u *SettingService) UpdatePort(port uint) error {
 				global.LOG.Errorf("restart system port failed, err: %v", err)
 			}
 		}()
-		masterAddr, err := settingRepo.Get(repo.WithByKey("MasterAddr"))
-		if err != nil {
-			global.LOG.Errorf("load master addr from db failed, err: %v", err)
-			return
-		}
-		if len(masterAddr.Value) != 0 {
-			oldMasterPort := loadPort(masterAddr.Value)
-			if len(oldMasterPort) != 0 {
-				if err := xpack.UpdateMasterAddr(strings.ReplaceAll(masterAddr.Value, oldMasterPort, fmt.Sprintf("%v", port))); err != nil {
-					global.LOG.Errorf("update master addr from db failed, err: %v", err)
-					return
-				}
-			}
-		}
 	}()
 	return nil
 }
```

Comment:
There does not appear to be any obvious issues with this code snippet from an English perspective. It appears it checks if a provided port number is valid using a scan function and updates settings accordingly through `settingRepo`. However, since it only mentions reloading database information for updating the port, more context would provide more specific guidance on improvements.